dspace-api/src/main/java/org/dspace/content/Community.java

```diff
@@ -264,4 +265,16 @@ private CommunityService getCommunityService() {
         return communityService;
     }
 
+    /**
+     * return count of the community items
+     *
+     * @return int
+     */
+    public int countArchivedItem() {
```

Comment:
Same here, this method should probably be plural `countArchivedItems` instead of `countArchivedItem`.

Reply:
Done.